### DISCUSSION TITLE
First Layer - Line Width

### DISCUSSION THREAD

**Discussion by JAM3SY** [OrcaSlicer/OrcaSlicer#4542] [Ideas]
Is it possible to add 'First Layer - Line Width' to Object process? Same as in Global process.

**Comment by TexZeTech** [comment]
I don't see why the devs couldn't do this. I would go request an enhancement and add your use case.

**Reply by JAM3SY** [reply]
Is there a dedicated 'request and enhancement' area in this github?

**Reply by TexZeTech** [reply]
1.) Issues
2.) New Issue
3.) 🚀 Feature Request / Enhancement
